main/lsp/LSPConfiguration.cc

```diff
@@ -84,6 +84,17 @@ LSPClientConfiguration::LSPClientConfiguration(const InitializeParams &params) {
                 clientHoverMarkupKind = getPreferredMarkupKind(contentFormat);
             }
         }
+        if (textDocument->codeAction) {
+            auto &codeAction = *textDocument->codeAction;
+            if (codeAction->dataSupport) {
+                clientCodeActionDataSupport = codeAction->dataSupport.value_or(false);
+            }
+
+            if (codeAction->resolveSupport) {
+                auto &properties = (*codeAction->resolveSupport)->properties;
+                clientCodeActionResolveEditSupport = std::count(properties.begin(), properties.end(), "edit") > 0;
```

Comment:
```suggestion
                clientCodeActionResolveEditSupport = absl::c_count(properties, "edit") > 0;
```

(This might require including `absl/algorithm/container.h` or somesuch.  Using `c_any_of` is also an option that would avoid the `> 0` test.)